src/components/useAuthContext.ts

```diff
@@ -0,0 +1,6 @@
+import { useContext } from 'react';
+import { AuthContext } from '../hooks/AuthContext';
+
+export const useAuthContext = () => {
+  return useContext(AuthContext);
+};
```

Comment:
better import it from the context folder, to avoid confusion. it isn't a component